app/com/yetu/oauth2provider/views/confirmedmail.scala.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+			<title>Successfully confirmed mail</title>
+			<link rel="stylesheet" media="screen" href="@controllers.routes.Assets.at("stylesheets/setup.css")">
+			<link rel="stylesheet" media="screen" href="@controllers.routes.Assets.at("stylesheets/confirmedmail.css")">
```

Comment:
It's better to use Assets.versioned, then it well be easier to replace css in production.